src/routers/catalog.ts

```diff
@@ -8,7 +8,7 @@ const eservicesRouter = zodiosRouter(api.api);
 eservicesRouter
```

Comment:
hai provato a farlo girare in locale con `tsc` dopo i refactor (a brevissimo mettiamo una CI!).
Per fargli vedere `req.authData`, va usato `ctx.router` anziché `zodiosRouter` altrimenti il type system non è aware del context custom che siamo iniettando

Reply:
ho applicato le modifica ad app.ts, creato da `zodiosContext` e passandolo alla funzione per la creazione del router con `ctx.router`, la tipizzazione dell'oggettao `authData` sembra funzionare  e tsc funziona.